scripts/test.sh

```diff
@@ -11,6 +11,9 @@ for f in $( git diff-tree $TRAVIS_COMMIT --name-status -r | grep php | grep -v "
 
 ./scripts/lint.sh
 
+export TERMINUS_CACHE_DIR="$HOME/.terminus/testcache"
+export TERMINUS_TOKENS_DIR="$HOME/.terminus/testcache/tokens"
```

Comment:
Nope. It's required. Tests break when I remove it. I'm guessing this is compiled before the env overrides are set:

https://github.com/pantheon-systems/terminus/blob/master/config/constants.yml#L31